powershell-adapter/powershell.resource.ps1

```diff
@@ -1,427 +1,349 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-
 [CmdletBinding()]
 param(
-    [ValidateSet('List','Get','Set','Test','Export','Validate')]
-    $Operation = 'List',
-    [Switch]
-    $WinPS = $false,
-    [Parameter(ValueFromPipeline)]
-    $stdinput
+    [Parameter(Mandatory = $true, Position = 0, HelpMessage = 'Operation to perform. Choose from List, Get, Set, Test, Export, Validate.')]
+    [ValidateSet('List', 'Get', 'Set', 'Test', 'Export', 'Validate')]
+    [string]$Operation = 'Default',
```

Comment:
Why is this `Default` and not one of the actual operations?

Reply:
the user should always explicitly choose. otherwise if something changes in the future, it could be a more severe breaking change.

Reply:
If `Mandatory=$true`, then the user has to specify.  Maybe just leave it `[string]$Operation` to avoid confusion

Reply:
fixed